chapters/io/ipc/drills/tasks/client-server/solution/Makefile

```
@@ -34,4 +34,9 @@ clean:
 	-rm -f $(OBJS) $(BINARIES)
 	-rm -f $(LOGGER)
 
+check:
+	chmod +x ../tests/check_client.sh ../tests/check_server.sh
+	../tests/check_client.sh
+	../tests/check_server.sh
```

```suggestion
check:
	../tests/check_client.sh
	../tests/check_server.sh
```
You can make the scripts executable already and that should be enough.